src/View/Board.elm

```diff
@@ -1,13 +1,13 @@
 module View.Board
     exposing
-        ( view
-        , config
+        ( config
+        , view
         )
 
-import Svg exposing (Svg, svg, g)
-import Svg.Attributes exposing (viewBox, style)
-import Svg.Events exposing (onClick)
 import Html exposing (Html)
+import Svg exposing (Svg, g, svg)
+import Svg.Attributes exposing (style, viewBox)
+import Svg.Events exposing (onClick)
```

Comment:
Is there a reason for the reordering of these lines? It's no big deal, just curious. Maybe is `elm-format` doing some stuff?